packages/fxa-shared/subscriptions/metadata.ts

```diff
@@ -59,12 +60,15 @@ export const metadataFromPlan = (plan: Plan): ProductMetadata => ({
  *     'product:name:xx-partial': true,
  *   },
  *
- * @param plan {Plan}
+ * @param plan Metadata fields from a subhub-esque Plan object
  * @param userLocales list of locale strings (only the first is used)
  * @returns {ProductDetails}
  */
 export const productDetailsFromPlan = (
-  plan: Plan,
+  plan: {
```

Comment:
Loosened up the type here: This function only needs a small subset of a Plan and this makes it easier to use in fxa-auth-server without type complaints.